.env.sample

```diff
@@ -29,11 +29,9 @@ REDIRECT_URIS='["http://localhost:8084/*","https://auth.dev.gke.otomi.cloud/*","
 
 # Gitea task
 GITEA_PASSWORD=''
-GITEA_URL='http://127.0.0.1:8082'
-DRONE_URL='http://127.0.0.1:8081'
-
-# Drone task
-DRONE_TOKEN=''
+GITEA_URL='http://127.0.0.1'
+GITEA_URL_PORT='8082'
+GITEA_OPERATOR_NAMESPACE='gitea-app'
```

Comment:
I would change this to gitea-app-operator so its more inline with the chart and other naming. Also gitea-app suggest to me that its a namespace were gitea apps live.